test/components/room_test.js

```diff
@@ -28,6 +28,37 @@ describe("Room component", () => {
         expect(roomComponent.find(StageProgressionButton)).to.have.length(1)
       })
     })
+
+    context("and there are no action items during the action-items stage", () => {
+      it("renders a disabled <StageProgressionButton>", () => {
+        const roomComponent = shallow(
+          <Room
+            {...defaultProps}
+            currentUser={facilitatorUser}
+            stage="action-items"
+          />
+        )
+        const stageProgressionButton = roomComponent.find(StageProgressionButton)
+        expect(stageProgressionButton).to.have.length(1)
```

Comment:
I think we can remove the `length(1)` expectations from both of these tests, as the StageProgressionButton is rendered in both cases